www/front_src/src/Header/SwitchThemeMode/index.tsx

```diff
@@ -1,105 +1,73 @@
+import { Switch, ListItemText } from '@mui/material';
 import makeStyles from '@mui/styles/makeStyles';
+import { grey } from '@mui/material/colors';
 
 import { userAtom, ThemeMode } from '@centreon/ui-context';
 import { patchData, useRequest } from '@centreon/ui';
 
-import svgSun from './images/sun.svg';
-import svgMoon from './images/moon.svg';
-
-interface StyleProps {
-  darkModeSvg?: string;
-  lightModeSvg?: string;
-}
-
-const ThemeModeSwitch = styled(Switch, {
-  shouldForwardProp: (prop) =>
-    !equals(prop, 'color') &&
-    !equals(prop, 'lightModeSvg') &&
-    !equals(prop, 'darkModeSvg'),
-})<StyleProps>(({ theme, darkModeSvg, lightModeSvg }) => ({
-  '& .MuiSwitch-switchBase': {
+const useStyles = makeStyles((theme) => ({
+  container: {
+    '& .MuiSwitch-thumb': {
+      backgroundColor: 'white',
+    },
+    '& .MuiSwitch-track': {
+      backgroundColor: '#aab4be',
+      opacity: 1,
+    },
+    alignItems: 'center',
+    display: 'flex',
+  },
+  containerMode: {
+    display: 'flex',
+    justifyContent: 'space-around',
+  },
+  containerSwitch: {
     '&.Mui-checked': {
-      '& + .MuiSwitch-track': {
-        backgroundColor: '#aab4be',
-        opacity: 1,
-      },
-      '& .MuiSwitch-thumb:before': {
-        backgroundImage: `url(${darkModeSvg})`,
+      '&:hover': {
+        backgroundColor: 'unset',
       },
-      color: 'transparent',
-      transform: 'translate(15px,-50%)',
     },
     '&:hover': {
-      backgroundColor: 'transparent',
+      backgroundColor: 'unset',
     },
-    color: 'black',
-    margin: 0,
-    position: 'absolute',
-    top: '50%',
-    transform: 'translate(-0.5px,-50%)',
   },
-  '& .MuiSwitch-thumb': {
-    '&:before': {
-      backgroundImage: `url(${lightModeSvg})`,
-      backgroundPosition: 'center',
-      backgroundRepeat: 'no-repeat',
-      content: "''",
-      height: '100%',
-      left: theme.spacing(0),
-      position: 'absolute',
-      top: theme.spacing(0),
-      width: '100%',
-    },
-    backgroundColor: 'white',
-    height: theme.spacing(3),
-    width: theme.spacing(3),
-  },
-  '& .MuiSwitch-track': {
-    backgroundColor: '#aab4be',
-    borderRadius: theme.spacing(10 / 8),
-    opacity: 1,
+  disabledMode: {
+    color: grey[600],
   },
-  height: theme.spacing(32 / 8),
-  padding: theme.spacing(11 / 8, 4 / 8, 11 / 8, 9 / 8),
-  width: theme.spacing(50 / 8),
-}));
-
-const useStyles = makeStyles((theme) => ({
-  container: {
-    alignItems: 'center',
-    display: 'flex',
-    paddingLeft: theme.spacing(7),
+  mode: {
+    paddingLeft: theme.spacing(1),
   },
 }));
 
 const SwitchThemeMode = (): JSX.Element => {
-  const props = {
-    darkModeSvg: svgMoon,
-    lightModeSvg: svgSun,
-  };
   const classes = useStyles();
   const { pathname } = useLocation();
-
   const { sendRequest } = useRequest({
     request: patchData,
   });
   const [user, setUser] = useAtom(userAtom);
-
+  const [, startTransition] = useTransition();
```

Comment:
I think we can use the isPending in order to disabled the switch. imo
```suggestion
  const [isPending, startTransition] = useTransition();
```